output.c

```diff
@@ -257,3 +321,62 @@ output_set_window_title(struct cg_output *output, const char *title)
 #endif
 	}
 }
+
+static bool
+output_config_apply(struct cg_server *server, struct wlr_output_configuration_v1 *config, bool test_only)
+{
+	struct wlr_output_configuration_head_v1 *head;
+	wl_list_for_each (head, &config->heads, link) {
+		struct cg_output *output;
+		bool ret = false;
+
+		wl_list_for_each (output, &server->outputs, link) {
```

Comment:
We don't need to iterate over the list of outputs here: we can grab the output via:

```c
struct cg_output *output = head->state.output->data;
```